style.css

```diff
@@ -84,4 +85,70 @@ form input {
         width: 1em;
     }
 
+}
+
+/* side nav bar */
+
+.earth-icon {
+    left: 0.2em;
+    bottom: 0.5em;
+}
+
+.info-icon {
+    right: 0.5em;
+    bottom: 2.7em;
+}
+
+.star-icon {
+    left: 0;
+    bottom: 0.7em;
+}
+
+.side-navbar {
+    height: 100vh;
+    width: 160px;
+    left: 0;
+    background-color: var(--white);
+    font-size: 15px;
+    overflow-x: hidden;
+    padding-top: 17px;
+    padding-left: 4em;
+    border-right: solid 1px var(--gray-4);
+}
+
+nav ul {
+    list-style-type: none;
+    display: inline;
+}
+
+.sidebar-item--active {
+    background-color: var(--gray-6);
+    border-right: solid 2px var(--primary-orange);
+    font-weight: 600;
+}
+
+.line-height-02 {
```

Comment:
Let's remove this class. It's not the right solution to aligning the icon. This is how the paragraph looks: 

<img width="244" alt="image" src="https://user-images.githubusercontent.com/11327761/200295369-1257ca35-3129-4be6-8dbd-cebf938a53c2.png">


It has no height. Find another way to position the icon.